fahrschein-example/src/main/java/org/zalando/fahrschein/example/Main.java

```diff
@@ -172,26 +174,8 @@ private static void subscriptionListen(ObjectMapper objectMapper, Listener<Sales
     }
 
     private static void subscriptionListenHttpComponents(ObjectMapper objectMapper, Listener<SalesOrderPlaced> listener) throws IOException {
-        final RequestConfig requestConfig = RequestConfig.custom()
-                .setSocketTimeout(60000)
-                .setConnectTimeout(2000)
-                .setConnectionRequestTimeout(8000)
-                .setContentCompressionEnabled(false)
-                .build();
-
-        final ConnectionConfig connectionConfig = ConnectionConfig.custom()
-                .setBufferSize(512)
-                .build();
 
-        final CloseableHttpClient httpClient = HttpClients.custom()
-                .setDefaultRequestConfig(requestConfig)
-                .setDefaultConnectionConfig(connectionConfig)
-                .setConnectionTimeToLive(30, TimeUnit.SECONDS)
-                .disableAutomaticRetries()
-                .disableRedirectHandling()
-                .setMaxConnTotal(8)
-                .setMaxConnPerRoute(2)
-                .build();
+        final CloseableHttpClient httpClient = HttpClients.createDefault();
```

Comment:
simplified this because without this code we can even run against some lower apache httpclient version.

Reply:
we should probably add some note about sizing the connection pooling correctly.

Reply:
So maybe just add this code as an example to the README file?

Reply:
done